src/cobalt.rs

```diff
@@ -23,14 +23,16 @@ pub fn build(source: &Path, dest: &Path, layout_str: &str, posts_str: &str) -> i
     // filename -> text content to the layout map
     for entry in walker.filter_map(|e| e.ok()).filter(|e| e.file_type().is_file()) {
```

Comment:
we already apply a filter on the files here, maybe you can create a filter function at the top level that checks if it's a file and if it doesn't start with ".", instead. We could also use that function in other places then.